packages/analytics/amplify_analytics_pinpoint/lib/src/device_context_info_provider/app_info_provider.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:package_info_plus/package_info_plus.dart';
+
+/// Data representation of AppInfo required for Pinpoint EndpointDemographic:
+/// https://docs.aws.amazon.com/pinpoint/latest/apireference/apps-application-id-endpoints.html
+class AppInfo {
+  /// The app name. `CFBundleDisplayName` on iOS, `application/label` on Android.
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

```suggestion
  /// The app name. 
  ///
  /// `CFBundleDisplayName` on iOS, `application/label` on Android.
```

Reply:
If this will be null on all other platforms, you could add that to the comment.